streams/src/test/java/org/apache/kafka/streams/kstream/internals/graph/StreamsGraphTest.java

```diff
@@ -129,7 +129,7 @@ public void shouldNotOptimizeWhenAThroughOperationIsDone() {
     public void shouldOptimizeSeveralMergeNodesWithCommonKeyChangingParent() {
         final StreamsBuilder streamsBuilder = new StreamsBuilder();
         final KStream<Integer, Integer> parentStream = streamsBuilder.stream("input_topic", Consumed.with(Serdes.Integer(), Serdes.Integer()))
-            .selectKey(Integer::sum);
+                                                                     .selectKey(Integer::sum);
```

Comment:
```suggestion
            .selectKey(Integer::sum);
```

Looks like an accidental formatting change.